web-client/src/presenter/actions/Login/submitLoginAction.tsx

```diff
@@ -12,17 +12,22 @@ export const submitLoginAction = async ({
 }> => {
   const { email, password } = get(state.authentication.form);
 
+  const cleanedPassword = password.trim();
```

Comment:
We are preventing all NEW passwords from having leading or trailing spaces, but this was only introduced 8 months ago. For everyone else they could have a leading space and we would be blocking them from logging in. In general although we are trying to help the user I think any modification of the user's input for the password is probably not a great idea, as it is not what they typed.

I am going to block this PR until we talk about it as I want to make sure we understand the consequences of this story.

Reply:
Thank you!

> but this was only introduced 8 months ago

This is great to know. I picked this ticket up, realized it might be tricky for reasons similar to what you outlined, put it back, got a comment from Tenille, and picked it back up.

Reply:
I am going to remove the block on this PR to allow you to continue forward! Thanks for chatting with everyone!